/**:
  ros__parameters:
    # default tracker models for each class
    car_tracker: "multi_vehicle_tracker"
    truck_tracker: "multi_vehicle_tracker"
    bus_tracker: "multi_vehicle_tracker"
    trailer_tracker: "multi_vehicle_tracker"
    pedestrian_tracker: "pedestrian_and_bicycle_tracker"
    bicycle_tracker: "pedestrian_and_bicycle_tracker"
    motorcycle_tracker: "pedestrian_and_bicycle_tracker"

    # default tracker node parameters
    publish_rate: 10.0
    world_frame_id: map
    enable_delay_compensation: false

    # debug parameters
    publish_processing_time: false
    publish_tentative_objects: false
    diagnostics_warn_delay: 0.5      # [sec]
    diagnostics_error_delay: 1.0     # [sec]
